src/core/validation/CustomValidationRules.ts

```diff
@@ -148,6 +148,15 @@ export class CustomValidationRules {
       },
       message: `${i18n.t('error_new_password_format')}`,
     })
+
+    extend('delHtmlTag', {
+      validate: (value) => {
+        const isHtmlTag = new RegExp(/<\/?.+?\/?>/g).test(value)
+        if (!isHtmlTag) return true
+        return false
```

Comment:
This is only doing **validation**.

Please also provide with a **transformer** for this.

Reply:
Are you sure that you don't want to just put validation on this one? This would avoid parsing the escaped text on the views @evias

Reply:
I don't think any tags should be used in the form. please let me know if you think it is necessary，thanks! @evias @decentraliser